### ISSUE TITLE
When did incremental became stable?

### ISSUE THREAD

**Issue by vicb** [opennextjs/opennextjs-aws#648]
@petebacondarwin noted this code in https://github.com/opennextjs/opennextjs-aws/blob/2202f36ce0f87357b249bd127cdd5e84d6deffd3/packages/open-next/src/build/createServerBundle.ts

```ts
  const isAfter141 =
    buildHelper.compareSemver(options.nextVersion, "14.0.4") >= 0;
// ...

    openNextReplacementPlugin({
      name: `utilOverride ${name}`,
      target: /core(\/|\\)util\.js/g,
      deletes: [
        ...(isAfter141
          ? ["experimentalIncrementalCacheHandler"]
          : ["stableIncrementalCache"]),
      ],
    }),
```

Which of 14.0.4 or 14.1 is the correct version?

**Comment by conico974** [comment]
Good catch, it's 14.1, even though technically it was released on a canary branch (i.e. v14.0.5-canary.65)

BTW you guys should probably throw on version of next < 13.4.13 (or at least show a big warning). 
Before that version, we use the built-in middleware/routing in Next (which will fail on cloudflare)
